lib/govuk_index/popularity_updater.rb

```diff
@@ -0,0 +1,62 @@
+module GovukIndex
+  class PopularityUpdater
+    SCROLL_BATCH_SIZE = 500
+    PROCESSOR_BATCH_SIZE = 25
+    TIMEOUT_SECONDS = 30
+
+    def self.update(index_name)
+      new(
+        source_index: index_name,
+        destination_index: index_name,
+      ).run
+    end
+
+    # This task is designed to migrate data when the schema changes, It waits for the
+    # queued jobs to be processed, this is done by monitoring the sidekiq queue and workers.
+    # This process should be closely monitored as it is not gauranteed to work.
+    def self.migrate(index_name)
+      raise('Please set `CONFIRM_INDEX_MIGRATION_START` to continue') unless ENV['CONFIRM_INDEX_MIGRATION_START']
```

Comment:
What's the purpose of this check, and should it be documented in the rake task too?